src/sentry/static/sentry/app/components/forms/apiForm.jsx

```diff
@@ -32,6 +38,11 @@ export default class ApiForm extends Form {
 
     let {data} = this.state;
 
+    // Excludes disabled fields
+    data = Object.keys(data)
```

Comment:
Can we refactor this to be a getter method on `<Form>` and make sure we access `this.state.data` in `<ApiForm>` and `<Form>` via the method in their `onSubmit` methods.